src/components/10-atoms/input-file/demo.js

```diff
@@ -0,0 +1,27 @@
+/* global document */
+import { storiesOf } from '@storybook/html';
+// import { html, render } from 'lit-html';
+import { withMarkdown } from '../../../../.storybook/addons/markdown';
+import Readme from './README.md';
+import './index';
+
+storiesOf('Atoms/Input File/Demos', module)
+  .addDecorator(withMarkdown(Readme))
+
+  .add('Feature - InputFile clickable', () => {
+    const inputFile = document.createElement('axa-input-file');
+    inputFile.setAttribute('multiple', 'multiple');
+    let counter = 0;
+    inputFile.innerHTML = `You have chosen ${counter} files`;
+
+    inputFile.addEventListener('input', e => {
+      const {
+        target: { files },
+      } = e;
```

Comment:
I'm not a fan of that. If you use that variable everywhere, why not. But using destructuring for only one place where is it being used makes the code too unreadable without any benefit imho. I leave it up to you if you want to change it or not, because this is very opinionated. But you need 3 lines of code that are completely unnecessary.

Reply:
it doesnt change that much but ok if you use it only once and the line you use it wont get to long makes no difference. but its up to you noah here